gateway-ha/src/main/java/com/lyft/data/gateway/ha/resource/PrestoResource.java

```diff
@@ -34,12 +33,13 @@
 
   @POST
   @Path("/resourcegroup/create")
-  public Response createResourceGroup(String jsonPayload) {
+  public Response createResourceGroup(@QueryParam("routingGroupDatabase")
```

Comment:
Do we want to rename this query param to something like ?useSchema=<schema-name>

Reply:
fixed..